.github/workflows/lintandformat.yml

```diff
@@ -17,15 +17,24 @@ jobs:
       # Setup Python (faster than using Python container)
       - name: Setup Python
         uses: actions/setup-python@v5
+        id: setup-python
         with:
           python-version: "3.11"
-          cache: "pip"
+
+      - name: Cache uv folder
+        id: cache-uv
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/uv
+          key: ${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-uv-${{ hashFiles('**/requirements.txt', '**/requirements-dev.txt') }}
+
+      - name: Install uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements-dev.txt
-          pip install "fastapi[all]"
+          uv pip install --system -r requirements-dev.txt
+          uv pip install --system "fastapi[all]"
```

Comment:
Why do we need to install faspapi[all] here ?

Reply:
I don't recall, @armanddidierjean may know